cypress/features/regression/carouselClassic.feature

```diff
@@ -7,12 +7,12 @@ Feature: Carousel component
   @positive
   Scenario: Initial slide title is Slide One
   # When step is run on background as Given
```

Comment:
Please add an information why this step is commented

Reply:
Amended comment to what the When step would actually say, as with other component feature files 👍

Reply:
Added "# commented because of BDD default scenario Given - When - Then"